frontend/src/scenes/session-recordings/SessionRecordingsTable.tsx

```diff
@@ -204,13 +204,19 @@ export function SessionRecordingsTable({ personUUID, isPersonPage = false }: Ses
                                     <span> {key}</span>
                                 </>
                             )}
-                            defaultValue="Last 30 days"
+                            defaultValue="Last 7 days"
                             bordered={true}
                             dateFrom={fromDate ?? undefined}
                             dateTo={toDate ?? undefined}
                             onChange={(changedDateFrom, changedDateTo) => {
                                 setDateRange(changedDateFrom, changedDateTo)
                             }}
+                            dateOptions={{
+                                Custom: { values: [] },
+                                'Last 24 hours': { values: ['-24h'] },
+                                'Last 7 days': { values: ['-7d'] },
+                                'Last 21 days': { values: ['-21d'] },
```

Comment:
Just curious, why 21 over 30?

Reply:
It's 21 days because we only keep recordings for that long.